packages/mdc-textfield/index.js

```diff
@@ -75,7 +78,37 @@ export class MDCTextfield extends MDCComponent {
           label.classList.remove(className);
         }
       },
-    }, this.getInputAdapterMethods_(), this.getHelptextAdapterMethods_()));
+    }, this.getInputAdapterMethods_(), this.getHelptextAdapterMethods_(), this.getBottomLineAdapterMethods_()));
+  }
+
+  getBottomLineAdapterMethods_() {
+    return {
+      addClassToBottomLine: (className) => {
+        if (this.bottomLine_) {
```

Comment:
The other option is to add an adapter method like: `isTextarea` and only register those when that adapter method returns false.